troubleshooting/known_issues/parry_peak_console_boot_errors.md

```diff
@@ -0,0 +1,77 @@
+# HPE Cray `EX255a` Boot Issue with Console Parameter
+
+- [Description](#description)
+- [Workaround](#workaround)
+
+## Description
+
+  `HPE Cray EX255a` hardware has an issue in it's firmware configuration that causes boots to stall when configured to use the default kernel serial port `ttyS0` which is provided in the compute image created during installation of CSM 1.5.2.
```

Comment:
`HPE Cray EX255a` hardware has an issue BIOS versions 1.1.0 and earlier which causes boots to stall when configured to use the default kernel serial port `ttyS0` which is provided in the compute image created during installation of CSM 1.5.2.